kratos/utilities/specifications_utilities.h

```diff
@@ -128,6 +134,28 @@ class KRATOS_API(KRATOS_CORE) SpecificationsUtilities
         ALE        = 2
     };
 
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /**
+     * @brief The default constructor
+     */
+    SpecificationsUtilities() = default;
+
+    /**
+     * @brief The default destructor
+     */
+    ~SpecificationsUtilities()= default;
+    
```

Comment:
the compiler does the same, no need to add this

Reply:
Done